doc/userguide/upgrade.rst

```diff
@@ -34,6 +34,27 @@ also check all the new features that have been added but are not covered by
 this guide. Those features are either not enabled by default or require
 dedicated new configuration.
 
+Upgrading to 7.0.8
+------------------
+- Unknown requirements in the ``requires`` keyword will now be treated
+  as unsatisfied requirements, causing the rule to not be loaded. See
+  :ref:`keyword_requires`. To opt out of this change and to ignore
+  uknown requirements, effectively treating them as satified the
+  ``ignore-unknown-requirements`` configuration option can be used.
```

Comment:
are you thinking about making this a 7.0.x only option? If so, we should probably state that this is not behavior that will continue in 8

Reply:
if we want to keep this in 8 and beyond, I'd like it to live under `detect.`

Reply:
Ideally we wouldn't have it at all. So will add a note that it won't be around in 8.

Reply:
> if we want to keep this in 8 and beyond, I'd like it to live under `detect.`

Also note the reasoning for the location: https://github.com/OISF/suricata/pull/12226#discussion_r1870844582